test/cases/auto_propagation/conf/uwsgi_auto.conf

```diff
@@ -12,7 +12,7 @@ http {
         listen       80;
 
         location / {
-            include /etc/nginx/uwsgi_params;
+            include /usr/local/openresty/nginx/conf/uwsgi_params;
```

Comment:
[suggestion] May I suggest another approach? Update the service [Dockerfile](https://github.com/DataDog/nginx-datadog/blob/master/test/services/nginx/Dockerfile) to create a symlink from `/usr/local/openresty/nginx/conf/uwsgi_params` to `include /etc/nginx/uwsgi_params`.